pandas/tests/indexes/period/test_arithmetic.py

```diff
@@ -11,7 +11,46 @@
 import pandas.core.indexes.period as period
 
 
+_common_mismatch = [pd.offsets.YearBegin(2),
+                    pd.offsets.MonthBegin(1),
+                    pd.offsets.Minute()]
+
+not_hourly_offsets = [timedelta(minutes=30),
```

Comment:
from here down should be fixtures, then you don't need the parameterize on each function, just pass the argument.

Reply:
just pushed with this change.